src/web/app/components/teammates-common/safe-html.pipe.ts

```diff
@@ -0,0 +1,21 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { DomSanitizer } from '@angular/platform-browser';
+
+/**
+ * Pipe to handle the display of raw HTML.
+ */
+@Pipe({
+  name: 'safeHtml',
+})
+export class SafeHtmlPipe implements PipeTransform {
+
+  constructor(private domSanitizer: DomSanitizer) {}
+
+  /**
+   * Transforms HTML to value that can be displayed without sanitization.
+   */
+  transform(html: string): any {
+    return this.domSanitizer.bypassSecurityTrustHtml(html);
```

Comment:
TinyMCE will generate inline CSS style for element. For some reasons, [innerHTML] will skip rendering the inline CSS style.

Reply:
I'm still quite wary because this creates an attack vector. Does `bypassSecurityTrustStyle` work?

Reply:
No. It does not work. `bypassSecurityTrustStyle` gives a style type while `[innerHTML]` expects a safe HTML.

![image](https://user-images.githubusercontent.com/13977235/52459239-f367bd00-2b9e-11e9-8a71-f9634ab3b23c.png)

> I'm still quite wary because this creates an attack vector.

CSP should work? (I have tried to inject some inline javascript and CSP does works)